test/docfx.Test/DocfxTest.cs

```diff
@@ -19,10 +19,12 @@ namespace Microsoft.Docs.Build
     public static class DocfxTest
     {
         private static readonly JsonDiff s_jsonDiff = CreateJsonDiff();
+        private static readonly JsonDiff s_languageServerNotificationJsonDiff = CreateLanguageServerNotificationJsonDiff();
```

Comment:
nit: this is likely to be reused for request also
```suggestion
        private static readonly JsonDiff s_languageServerJsonDiff = CreateLanguageServerJsonDiff();
```